spec/system/support_interface/docs_spec.rb

```diff
@@ -31,6 +31,7 @@ def and_it_contains_documentation_for_all_emails
     emails_outside_of_states = %w[
       authentication_mailer-sign_in_email
       authentication_mailer-sign_in_without_account_email
+      candidate_mailer-all_application_choices_rejected
```

Comment:
This can be documented here instead:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/config/locales/application_states.yml#L204-L209